pkg/snapshot/config.go

```diff
@@ -16,43 +16,71 @@
 package snapshot
 
 import (
+	ethNode "github.com/ethereum/go-ethereum/statediff/indexer/node"
+	"github.com/ethereum/go-ethereum/statediff/indexer/postgres"
 	"github.com/spf13/viper"
-
-	"github.com/vulcanize/ipfs-blockchain-watcher/pkg/config"
-	"github.com/vulcanize/ipfs-blockchain-watcher/pkg/core"
 )
 
 const (
-	ANCIENT_DB_PATH   = "ANCIENT_DB_PATH"
-	ETH_CLIENT_NAME   = "ETH_CLIENT_NAME"
-	ETH_GENESIS_BLOCK = "ETH_GENESIS_BLOCK"
-	ETH_NETWORK_ID    = "ETH_NETWORK_ID"
-	ETH_NODE_ID       = "ETH_NODE_ID"
-	LVL_DB_PATH       = "LVL_DB_PATH"
+	ancientDBPath   = "ANCIENT_DB_PATH"
```

Comment:
I prefer to keep these variable names as they previously were, we have an ongoing convention of using this format for env variables. I know linters/IDEs sometimes complain but it's nice imo because 1) you can tell right away the Go variable represents an ENV variable and additionally 2) you know exactly what is the ENV variable the Go variable represents just looking at the Go variable name. I've seen this convention elsewhere as well.

Reply:
Done.